admin/src/app/produtores/page.tsx

```diff
@@ -1,5 +1,49 @@
+"use client";
+
+import { useEffect, useState } from "react";
+
+import IProducerTable from "../components/ProducerTable";
```

Comment:
Interfaces com o prefixo I não serão mais utilizadas após o merge do PR #169. Será necessário atualizar quando ele for aprovado.